documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -728,6 +728,24 @@ minIdle = 1
 ----
 ====
 
+The `hotrodclient.properties` is optional, in this case Hibernate OGM will use the following:
+
+|===
+| HotRod client property | Hibernate OGM mandatory value
+| infinispan.client.hotrod.marshaller
+| org.hibernate.ogm.datastore.infinispanremote.impl.protostream.OgmProtoStreamMarshaller
+| infinispan.client.hotrod.force_return_values
+| true
+|===
+
+Every other property will use the default values defined by http://infinispan.org/docs/stable/user_guide/user_guide.html#hotrod:java-client[Infinispan - Java Hot Rod client].
+
+[IMPORTANT]
+====
+These values for these properties are mandatory in order to keep the expected behaviour of the dialect.
```

Comment:
This warning could be clearer if you could state explicitly that you are referring to the above two properties, or maybe even repeat them one last time?

Reply:
Good idea. I will change it in that way